src/Bench/JsonSerializableBenchmark.php

```diff
@@ -22,4 +22,9 @@ public function serialize($data): void
             $data
         );
     }
+
+    public function getPackageName(): ?string
+    {
+        return null;
```

Comment:
here could be `php/json_serialize`

Reply:
But this package will not be found on `package.json`, right? What should be displayed as output?

Reply:
see https://github.com/php-serializers/ivory-serializer-benchmark/pull/13#discussion_r234678021